src/lib/final_report.ml

```diff
@@ -427,7 +427,7 @@ module To_workflow
               (get_optitype_result i)#product#path);
           "Vaxrank",
           Option.map vaxrank ~f:(fun i ->
-              (get_vaxrank_result i)#product#text_report_path);
+              (get_vaxrank_result i)#product#output_folder_path);
```

Comment:
yeah - this is a temporary solution just to make epidisco compile. I believe @ihodes is cooking a better solution around this (#105), so I didn't want to re-implement that in a different branch.